src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -10,30 +10,15 @@ Microsoft.CodeAnalysis.MetadataImportOptions
 Microsoft.CodeAnalysis.MetadataImportOptions.All = 2 -> Microsoft.CodeAnalysis.MetadataImportOptions
 Microsoft.CodeAnalysis.MetadataImportOptions.Internal = 1 -> Microsoft.CodeAnalysis.MetadataImportOptions
 Microsoft.CodeAnalysis.MetadataImportOptions.Public = 0 -> Microsoft.CodeAnalysis.MetadataImportOptions
+Microsoft.CodeAnalysis.OperationKind.CaughtException = 94 -> Microsoft.CodeAnalysis.OperationKind
```

Comment:
In this file, I would expect to see only changes around IInstanceReferenceOperation.ReferenceKind property and InstanceReferenceKind type. Please revert all other unnecessary changes.

Reply:
The codefix that adds things to the public api automatically sorts this file. If we don't do this now, we'll have to manually fix this file up every time we want to add anything new. I'm going to leave this as is.

---
In reply to: [180511221](https://github.com/dotnet/roslyn/pull/26055#discussion_r180511221) [](ancestors = 180511221)

Reply:
If the sorting is necessary, we can apply it in a separate targeted PR.

---
In reply to: [180514762](https://github.com/dotnet/roslyn/pull/26055#discussion_r180514762) [](ancestors = 180514762,180511221)

Reply:
Given that this was a generated change, I'm going to leave it as is. I did not manually change this file at all, this is purely codefix.

---
In reply to: [180522270](https://github.com/dotnet/roslyn/pull/26055#discussion_r180522270) [](ancestors = 180522270,180515588,180514762,180511221)